kitsune/questions/config.py

```diff
@@ -34,37 +34,37 @@
                         # TODO: Just use the IA topics for this.
                         # See bug 979397
                         (
-                            "download-and-install",
+                            "Install and Update",
```

Comment:
This should be the `install-and-update` slug instead, since it'll be attached to an `<option>` element as a `value` attribute right?